modules/Reports/templates_assets.php

```diff
@@ -108,6 +133,11 @@
                     return;
             }
 
+            $actions->addAction('edit', __('Change Active Status'))
+                ->setIcon('cross')
+                ->setURL('/modules/Reports/templates_assets_edit.php')
+                ->addParam('gibbonReportPrototypeSectionID', $template['gibbonReportPrototypeSectionID']);
+
```

Comment:
Hmmm, this is interesting to add it as a stand-alone action, rather than part of the edit action? 🤔